tests/documentation/DocumentationStyleBearTest.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+from bears.documentation.DocumentationStyleBear import DocumentationStyleBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+def load_testfile(test, filename):
+    filepath = os.path.join(os.path.dirname(__file__), "test_files",
+                            "DocumentationStyleBear", filename)
+    with open(filepath) as fl:
+        return fl.read()
+
+good_file = load_testfile("DocumentationStyleBear", "good_file.py")
+bad_file = load_testfile("DocumentationStyleBear", "bad_file.py")
+
+file_sameline = load_testfile("DocumentationStyleBear", "bad_file2.py")
```

Comment:
I think using files here makes sense for the sake of clean code. Because I am going to be testing a lot of files of different languages in the future.. 

CC @Makman2, what do you think?